fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml

```diff
@@ -82,4 +82,5 @@
     <include file="parts/UC-68-additional-guarantor-information.xml" relativeToChangelogFile="true" />
     <include file="parts/OXY-116_BNPL_loan_product_configuration.xml" relativeToChangelogFile="true" />
     <include file="parts/OXY-118-BNPL_attributes_for_loan.xml" relativeToChangelogFile="true" />
+    <include file="parts/0042_add_product_categories_and_product_type_column.xml" relativeToChangelogFile="true" />
```

Comment:
Please add the jira ticket number to this at least to allow us move this to a custom folder when we need to.
Also i think you should configure your IDE to ignore formatting of code that is not added by you. Your diff is growing so big with work from other developers

Reply:
Hi have changed the migration name @Napho .

I did ./gradlew :fineract-provider:spotlessApply and commited @hagafrank , but nothing changed. I think I need to follow what Naphlin is sugesting next time